### PR TITLE
compete rebuild - first step for plugins

### PR DESCRIPTION
**Pull request by stevermeister**

### CODE REVIEW

**Comment by stevermeister** [comment]
@bampakoa you are welcome to review.
I think we need to merge this step and think about plugins structure after.

File: package.json [hunk 1]
```diff
@@ -1,80 +1,53 @@
...
-    "docs:build": "compodoc -p tsconfig.json -n ngx-wig -d docs --hideGenerator",
-    "docs:serve": "npm run docs:build -- -s",
-    "docs:watch": "npm run docs:build -- -s -w",
-    "lint": "tslint --type-check --project tsconfig.json src/**/*.ts",
-    "lite": "lite-server",
-    "playground:build": "tsc -p playground -w",
-    "playground": "concurrently \"npm run build:watch\" \"npm run playground:build\" \"npm run lite\"",
-    "test": "tsc && karma start"
+    "lib-build": "ng build ngx-wig",
+    "start": "(ng build ngx-wig --watch &) && sleep 10 && ng serve",
```
Comment (bampakoa) [inline package.json]: Is additional `&` a typo?

File: projects/ngx-wig/src/lib/ngx-wig.module.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA, ModuleWithProviders, InjectionToken } from '@angular/core';
```
Comment (bampakoa) [inline projects/ngx-wig/src/lib/ngx-wig.module.ts]: `CUSTOM_ELEMENTS_SCHEMA` and `NO_ERRORS_SCHEMA` are unused

File: projects/ngx-wig/src/lib/ngx-wig.module.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+@NgModule({
+  declarations: [
+    NgxWigComponent
+  ],
+  imports: [
+    BrowserModule,
+    FormsModule,
+    ReactiveFormsModule,
```
Comment (bampakoa) [inline projects/ngx-wig/src/lib/ngx-wig.module.ts]: Should we let the consumer of the library control this?

File: src/app/app.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { TestBed, async } from '@angular/core/testing';
```
Comment (bampakoa) [inline src/app/app.component.spec.ts]: We could omit the whole file. No need to test the playground :smiley:

File: package.json (hunk 1 above)
Comment (yuriykuzin) [reply package.json]: @bampakoa nope, that's for parallel run. We run lib build watch and leave it running. And meanwhile waiting 10 secs before running `ng serve`

File: projects/ngx-wig/src/lib/ngx-wig.module.ts (hunk 2 above)
Comment (stevermeister) [reply projects/ngx-wig/src/lib/ngx-wig.module.ts]: yes, true

File: src/app/app.component.spec.ts (hunk 4 above)
Comment (stevermeister) [reply src/app/app.component.spec.ts]: also true :)

File: projects/ngx-wig/src/lib/ngx-wig.module.ts (hunk 3 above)
Comment (stevermeister) [reply projects/ngx-wig/src/lib/ngx-wig.module.ts]: I thought that we are using it, but it looks like you are right, removed

File: package.json (hunk 1 above)
Comment (bampakoa) [reply package.json]: @yuriykuzin is this OS agnostic?

**Comment by stevermeister** [comment]
published!

**Pull request merged**